bin/codecept.js

```diff
@@ -155,6 +156,20 @@ program.command('run-multiple [suites...]')
 
   .action(require('../lib/command/run-multiple'));
 
+program.command('info')
+  .description('Print debugging information concerning the local environment')
+  .action(() => {
+    console.log('\n Environment information:-\n');
+    envinfo
+      .run({
+        System: ['OS', 'CPU'],
+        Binaries: ['Node', 'Yarn', 'npm'],
+        Browsers: ['Chrome', 'Edge', 'Firefox', 'Safari'],
+        npmGlobalPackages: ['codeceptjs'],
+      })
+      .then(console.log);
```

Comment:
do we really need this?

Reply:
Are you asking about browser information?

Reply:
no, the `.then(console.log);` line

Reply:
Logs the respective data (env-info) as [here](https://github.com/Codeception/CodeceptJS/pull/1839#issue-306217645)
It returns a `Promise`